src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -1155,4 +1155,18 @@ private function createSubCrawler($nodes)
 
         return $crawler;
     }
+
+    private function triggerDeprecation($methodName)
+    {
+        $traces = debug_backtrace();
```

Comment:
getting the debug backtrace should be done only on HHVM, to avoid extra work for PHP